internal/rollup/terser-wrapped.js

```diff
@@ -20,47 +20,81 @@
 // each .js file in that folder and outputs to a specified output
 // folder. If the input is a file then terser is just run
 // one that individual file.
-
 const fs = require('fs');
 const path = require('path');
-const tmp = require('tmp');
-const child_process = require('child_process');
 
-const DEBUG = false;
+// worker_threads is only avilable on version > 10.5.0
+const [major, minor] = process.version.replace('v', '').split('.');
+const WORKER_THREADS_AVILABLE = major > 10 || major == 10 && minor > 5
```

Comment:
nit: why not just try/catch on the `require` call?

Reply:
should also be `minor >= 5`

Reply:
Total guess here, but I would assume that doing arithmetic is faster than recovering from an exception.
However, in hindsight, it's probably the smallest amount of performance difference here.